app.rb

```diff
@@ -1,9 +1,30 @@
 require 'sinatra/base'
+require_relative './lib/peeps'
+require_relative './lib/peep'
+require_relative './lib/peeps_data'
```

Comment:
This looks good.  Can you find a way of removing the global var?  Also, please be sure to revisit RESTful routing.

The restful routes for this app would be
```
get '/peeps' # to get all the peeps and render them in the :index view
get '/peeps/new' # to get the form which is used to create a new peep
post '/peeps' # to handle form submission
```